include/odp/api/spec/packet_io.h

```diff
@@ -412,6 +412,20 @@ typedef union odp_pktout_config_opt_t {
 		/** Insert SCTP checksum on packet by default */
 		uint64_t sctp_chksum     : 1;
 
+		/** Packet references not used on packet output
+		 *
+		 * When set, application indicates that it will not transmit
+		 * packet references on this packet IO interface.
+		 * Since every ODP implementation supports it, it is always
+		 * ok to set this flag when needed.
```

Comment:
I would like to have the "when needed" part removed as it implies that the flag is sometimes _needed_ and in (only?) those particular cases it is ok to set it. But the flag is always optional and never strictly needed.